test/e2e-tests-upgrade.sh

```diff
@@ -46,12 +46,15 @@ set +o pipefail
 header "Install the previous release of Tekton pipeline $PREVIOUS_PIPELINE_VERSION"
 install_pipeline_crd_version $PREVIOUS_PIPELINE_VERSION
 
+failed=0
+# The integration test for the older version to prevent regression on existing changes.
+go_test_e2e -timeout=20m ./test -run ^TestUpgradeAgainstPreviousVersion || failed=1
```

Comment:
When I run this locally I see
```
Running tests with 'go test -race -v  -count=1 -tags=e2e -timeout=20m ./test -run ^TestUpgradeAgainstPreviousVersion '
=== RUN   TestUpgradeAgainstPreviousVersion
=== PAUSE TestUpgradeAgainstPreviousVersion
=== CONT  TestUpgradeAgainstPreviousVersion
    upgrade_test.go:45: Create namespace arendelle-hbk4z to deploy to
    upgrade_test.go:45: Failed to get ConfigMap `config-defaults`: configmaps "config-defaults" not found
--- FAIL: TestUpgradeAgainstPreviousVersion (0.21s)
```

Does this work?

Reply:
Looking at the logs, the `config-defaults` should be able to get found with SYSTEM_NAMESPACE being set to tekton-pipelines. It has only been set in `install_pipeline_crd` previously, I refactored at 
https://github.com/tektoncd/pipeline/pull/6901.

Reply:
thanks! Since this PR needs SYSTEM_NAMESPACE to work correctly, I think you can go ahead and just set that environment variable in this PR.

Reply:
SG, updated in this PR